modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/internal/WFSRequestTest.java

```diff
@@ -0,0 +1,110 @@
+package org.geotools.data.wfs.internal;

```

Comment:
header required, thought automatic code checks would catch this?

Reply:
You mean, wrong location? The header is under the imports. I can change that.